v3.1/glfw/glfw/deps/EGL/dummy.go

```diff
@@ -0,0 +1,5 @@
+//+build ignore
```

Comment:
There should be a space between `//` and `+build`:

```
// +build ignore
```

For consistency with the Go project.